odl/test/largescale/tomo/analytic_slow_test.py

```diff
@@ -40,14 +40,22 @@
               skip_if_no_astra_cuda('helical astra_cuda uniform'),
               skip_if_no_skimage('par2d skimage uniform')]
 
-projector_ids = [" geom='{}' - impl='{}' - angles='{}' "
-                 ''.format(*p.args[1].split()) for p in projectors]
-
+projector_ids = [
+    " geom='{}' - impl='{}' - angles='{}' ".format(*p.values[0].split())
+    for p in projectors
+]
 
 # bug in pytest (ignores pytestmark) forces us to do this this
 largescale = " or not pytest.config.getoption('--largescale')"
-projectors = [pytest.mark.skipif(p.args[0] + largescale, p.args[1])
-              for p in projectors]
+projectors = [
```

Comment:
Is this truly needed? It's honestly hard to read.

Reply:
It ain't pretty, I agree. Maybe I'll just make a little function to do the dirty work.

Formatting-wise, I'm currently going with what the [`black`](https://github.com/ambv/black) tool spits out. It usually does a good job and makes code formatting a no-brainer. Sometimes it looks a bit like Lisp. Is that a bad thing? :stuck_out_tongue:

Reply:
Lets go ahead for now, but long term I'm not sure if it is a bit to verbose

Reply:
Actually that code didn't work at all, as I found out. I changed the implementation, now I think it's much cleaner. Please check again.